packages/framework/src/Model/Product/Search/Export/ProductSearchExporter.php

```diff
@@ -53,7 +115,42 @@ public function export(int $domainId, string $locale): void
             $exportedIds = array_merge($exportedIds, $batchExportedIds);
             $startFrom += static::BATCH_SIZE;
         } while (!empty($batchExportedIds));
-        $this->removeNotUpdated((string)$domainId, $exportedIds);
+        $this->removeNotUpdated($domainId, $exportedIds);
```

Comment:
is this change intentional? It is in deprecated function...

Reply:
Yes because I added strict types notation and this would be broken.